asf_levies_model/summary.py

```diff
@@ -0,0 +1,227 @@
+import pandas as pd
+from typing import Dict, Optional
+
+
+def _sum_levies(val: float, summary: str, fuel: str, levies: list) -> float:
+    """Calculate sum of levies."""
```

Comment:
Originally I didn't give this a full docstring because it's an internal function. However, I think you're right that we're likely to want to reuse it as the model evolves, so this provided docstring is much appreciated!